views/js/qtiCreator/widgets/interactions/selectPointInteraction/states/Map.js

```diff
@@ -196,7 +196,14 @@ define([
                 } else {
                     //move the score back the shape and show the popup
                     shape.toFront();
-                    $('#score-popup-' + shape.id).show(); 
+                    $('#score-popup-' + shape.id).show();
+
+                    //the click on the cross hides the popup
+                    $('.mapping-editor').on('click', '.closer', function(){
+                        $('#score-popup-' + shape.id).hide();
+                    });
+
+                    tooltip.lookup($('.panel'));
```

Comment:
It appears to be not working.
@handleman something forgotten during the tooltip replacement ?

Reply:
This tooltip was not initialized anywhere.
in the import there was an old 'ui / tooltipster' and I replaced it with the actual 'ui / tooltip' and initialized it.
works for me.
maybe he didn't notice.